my_list.rb

```diff
@@ -0,0 +1,18 @@
+require_relative('my_enumerable')
+class MyList
+  include MyEnumerable
+  def initialize(list)
+    @list = list
+  end
+
+  def each(&block)
+    @list.each { |a| block.call a }
+  end
```

Comment:
- [ ] when creating the `each` method please avoid using the already existing method.

Reply:
Changed to yield